src/backends/custom/custom_backend.cc

```diff
@@ -583,28 +583,24 @@ CustomBackend::Context::GetOutput(
     if (shape_dim_cnt > 0) {
       shape.assign(shape_dims, shape_dims + shape_dim_cnt);
     }
+
+    auto actual_memory_type = ToTRTServerMemoryType(*memory_type);
```

Comment:
Don't initialize actual_memory_type... just causes overhead and it is an "out" parameter,  not an "inout"

Reply:
Done. Moved this initialization inside the function.